patch-package
new file mode 100644
Binary files /dev/null and b/node_modules/electron-builder/.DS_Store differ
new file mode 100644
Binary files /dev/null and b/node_modules/electron-builder/node_modules/.DS_Store differ
--- a/node_modules/electron-builder/node_modules/electron-osx-sign/sign.js
+++ b/node_modules/electron-builder/node_modules/electron-osx-sign/sign.js
@@ -156,14 +156,15 @@ function signApplicationAsync (opts) {
       if (opts.timestamp) {
         args.push('--timestamp=' + opts.timestamp)
       }
-      if (opts.hardenedRuntime || opts['hardened-runtime']) {
+//      if (opts.hardenedRuntime || opts['hardened-runtime']) {
         // 17.7.0 === 10.13.6
         if (compareVersion(osRelease, '17.7.0') >= 0) {
           args.push('--options', 'runtime')
+          console.log("Enabling hardened runtime");
         } else {
           debuglog('Not enabling hardened runtime, current macOS version too low, requires 10.13.6 and higher')
         }
-      }
+//      }
 
       var promise
       if (opts.entitlements) {
